docs/platforms/javascript/guides/nuxt/install/cli-import.mdx

```diff
@@ -7,17 +7,17 @@ description: "Learn how to use the node --import CLI flag."
 ## Understanding the `--import` CLI Flag
 
 The [`--import` CLI flag](https://nodejs.org/api/cli.html#--importmodule) in Node is the default way in ESM to preload a module before the application starts.
-If you cannot use the SDK's <PlatformLink to="/install/dynamic-import/">default dynamic import behaviour</PlatformLink>, setting
-this flag is crucial for ensuring proper server-side initialization, as Sentry needs to be initialized before the rest of the application runs.
+Setting this flag is crucial for ensuring proper server-side initialization, as Sentry needs to be initialized before the rest of the application runs.
 
 ## Scenarios where `--import` does not work
 
-- You're not able to add Node CLI flags or environment variables that are scoped to the function runtime
+- You're not able to add Node CLI flags or environment variables scoped to the function runtime, meaning these variables are not applied in other scopes, such as build time.
```

Comment:
```suggestion
- You're not able to add Node CLI flags or environment variables scoped to the function runtime, meaning these variables are not applied in other scopes, such as at build time.
```

Reply:
I think we shouldn't use abbreviations like e.g. but let's see what Liza has to say :)

Reply:
@s1gr1d is right :) we avoid latin abbreviations